src/servers/ZoneServer/zoneserver.ts

```diff
@@ -1931,6 +1982,70 @@ export class ZoneServer extends EventEmitter {
   spawnProps(client: Client): void {
     this.spawnNpcCollection(client, this._props);
   }
+  
+  customizeDTO(client: Client): void {
+    const DTOArray: any = [];
```

Comment:
i saw that "any" is used a lot maybe we could make a type / class for dto ?

Reply:
I will do that in another PR, we need classes for npcs also and the dto class will inherit from it